components/styles.js

```diff
@@ -76,16 +75,19 @@ const styles = StyleSheet.create({
     backgroundColor: 'transparent',
     width: 28,
     height: 28,
+    left: 13,
+    top: 13,
     borderRadius: 14,
+    position: 'absolute',
```

Comment:
Not working on centering it without absolute position right now, but yes, that's the goal.
I'm trying to figure out the black dayTextStyle...this makes no sense.